src/Objs/Job/JobComponent.js

```diff
@@ -7,17 +7,22 @@ import SchemaDotOrg from '../../Components/SchemaDotOrg';
 Scrivito.provideComponent('Job', ({ page }) => {
   return (
     <div>
-      <section className="bg-white">
+      <section className="title-bgc">
         <div className="container">
-          <JobDatePosted datePosted={page.get('datePosted')} />
-          <Scrivito.ContentTag tag="h1" className="h2" content={page} attribute="title" />
-          <JobLocation job={page} />
-          <JobValidThrough validThrough={page.get('validThrough')} />
-          <JobEmploymentTypes employmentTypes={page.get('employmentType')} />
+          <div className="row">
```

Comment:
just add class 'align-items-center' to row ... so you don't need  any 'd-flex' and 'align-self-center' at 'col-lg-7' AND 'col-lg-5'